src/components/structures/RoomSubList.js

```diff
@@ -78,6 +78,9 @@ var RoomSubList = React.createClass({
         // undefined if no room is selected (eg we are showing settings)
         selectedRoom: React.PropTypes.string,
 
+        // pass explicitly, do not rely on label==='Invites' because i18n.
```

Comment:
Out of context, this comment doesn't mean much. Maybe just mention that it's passed through to RoomTile and makes the header highlighted regardless of notifications.